src/TestingUtils/Microsoft.AspNetCore.Testing/src/xunit/FlakyAttribute.cs

```diff
@@ -50,7 +50,7 @@ namespace Microsoft.AspNetCore.Testing.xunit
     /// </para>
     /// </example>
     [TraitDiscoverer("Microsoft.AspNetCore.Testing.xunit.FlakyTestDiscoverer", "Microsoft.AspNetCore.Testing")]
-    [AttributeUsage(AttributeTargets.Method)]
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly)]
```

Comment:
Need a test covering at least one of the newly supported uses of the attribute

Reply:
We disabled all the FlakyAttribute tests because of the noise they produced. I don't think it's a good idea to bring them back.